target/linux/generic/files/drivers/net/phy/rtl8367.c

```diff
@@ -1077,21 +1077,37 @@ static int rtl8367_led_blinkrate_set(struct rtl8366_smi *smi, unsigned int rate)
 }
 
 #ifdef CONFIG_OF
-static int rtl8367_extif_init_of(struct rtl8366_smi *smi, int id,
+static int rtl8367_extif_init_of(struct rtl8366_smi *smi,
 				 const char *name)
 {
 	struct rtl8367_extif_config *cfg;
 	const __be32 *prop;
 	int size;
 	int err;
+	unsigned cpu_port;
+	unsigned id = UINT_MAX;
 
 	prop = of_get_property(smi->parent->of_node, name, &size);
-	if (!prop)
-		return rtl8367_extif_init(smi, id, NULL);
```

Comment:
You removed the support without configuration. Are we using configurations in all device tree files using the realtek driver?

Reply:
The (!prop) case is handled in line 1091

Reply:
The (!prop) case is handled in line 1091
There is a configuration for all devices with rtl8367